quickstarts/android/03-session-handling.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Auth0 Android Quickstarts - 3. Session Handling
+description: This tutorial will show you how to use Lock10 to mantain a session connected.
+---
+
+## Android - Session Handling Tutorial
+
+This tutorial will show you how to use Lock10 to mantain an active session with Auth0.
+
+The user's Credentials, contains three kind of tokens:
+
+* idToken: Identity Token that proves the identity of the user.
+* accessToken: Access Token used by Auth0 API.
+* refreshToken: Refresh Token that can be used to request new tokens without signing in again.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+
+
+### Before Starting
+
+Be sure that you have completed the [Basic Login](01-login.md) Quickstart.
+
+### 1. Save the User's Credential
+
+Your first step is to save, through a secure method, the user's credentials obtained in the login success response. We won't cover how to do that on this tutorial, feel free to save it.
+
+```Android
+private LockCallback callback = new AuthenticationCallback() {
+            @Override
+            public void onAuthentication(Credentials credentials) {
+				// Login Success response
+				saveCredentials(credentials)
+            }
+				...
+        };
+```
+
+ 
+### 2. Request a new tokenID
+
+#### i. Using a non-expired tokenID
+
+In the case your current tokenID didn't expire, you can use it to get a new one, with a newer expire date.
+
+        String idToken = // GET STORED TOKEN ID
+        AuthenticationAPIClient client = new AuthenticationAPIClient(new Auth0(CLIENT_ID, AUTH0_DOMAIN));
+        
+        client.delegationWithIdToken(idToken).start(new BaseCallback<Delegation>() {
+            @Override
+            public void onSuccess(Delegation payload) {
+            
+            payload.getIdToken(); // New ID Token
+            payload.getExpiresIn(); // New ID Token Expire Date
+            
+            }
+
+            @Override
+            public void onFailure(Auth0Exception error) {
+
+            }
+        });
+                
+
+#### ii. Using refreshToken
+
+If the tokenID already expired, you can always use the refreshToken to get a new one, without having to login again. For this reason, this token must be securely saved.
```

Comment:
Again, better to backtick `tokenID` and `refreshToken` here. Please, check all their appearances in the file and see if it's convenient to add backticks to them (it is, most of times).